packages/js/product-editor/src/blocks/pricing/README.md

```diff
@@ -0,0 +1,64 @@
+# woocommerce/product-pricing-field
+
+A product price block with currency display.
+
+![Product pricing field](https://woocommerce.files.wordpress.com/2023/09/woocommerceproduct-pricing-field.png)
+
+## Attributes
+
+### property
+
+- **Type:** `String`
+- **Required:** `Yes`
+
+Property in which the price value is stored.
+
+### label
+
+- **Type:** `String`
+- **Required:** `Yes`
+
+Label that appears at the side of the price field.
```

Comment:
It's totally nitpicky, but in the example the label appears on top of the price field.

Reply:
that's not nitpicky at all. Thanks for catching it.